src/__tests__/ioredis-passed-in.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, it } from "vitest";
+import { resumableStreamTests } from "./tests";
+import Redis from "ioredis";
+
+if (process.env.REDIS_URL) {
+  resumableStreamTests(() => {
+    return {
+      subscriber: new Redis(process.env.REDIS_URL!),
+      publisher: new Redis(process.env.REDIS_URL!),
+    };
+  }, "ioredis");
```

Comment:
I added an extra modifier to the tests here to test the different entry points for the app. For instance, this one still tests the ioredis entry point.